MVP/MVP/MVP/ViewModels/MyProfileViewModel.cs

```diff
@@ -75,7 +76,7 @@ public static MyProfileViewModel Instance
         private string _strMeImage = "Me.png";
         private string _strNotificationsImage = "Notifications.png";
         private string _strSupportImage = "Support.png";
-        private string _strSettingsImage = "Settings.png";
+        private string _strSettingsImage = "icon_settings.png";
```

Comment:
I'm not even sure why these are being bound. Looks like its done to have the same image be culture specific, but there are better ways to do that I'd say (e.g. a ValueConverter). For cleanliness sake I wouldn't want this kind of stuff in the business part (where it is now) since it's a UI thing.

Reply:
I added a new document, we should to through ALL of the images, rename them and then do a OnPlatform for UWP for the Assets folder where they actually belong.

Reply:
I agree, they should NOT be in there at all (the viewmodels that is). It doesn't feel right and isn't necessary.